2021-offseason/src/main/java/frc/robot/subsystems/Drivetrain.java

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
+import com.revrobotics.CANSparkMax;
+import edu.wpi.first.wpilibj.drive.DifferentialDrive;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+
+public class Drivetrain extends SubsystemBase {
+  private final CANSparkMax frontLeft, frontRight, backLeft, backRight;
+  private final DifferentialDrive difDrive;
+  private final boolean driveInverted;
```

Comment:
does this need to be a member variable? it's only used in the constructor
furthermore, do we need to make this its own variable? you could just do `frontLeft.setInverted(!Constants.kDrivetrain.DRIVE_INVERTED);` and I don't think that would change anything or decrease code readability

Reply:
i'm pretty sure this already is a constant in this pr ?

Reply:
but also we usually have a method within the drive subsystem to change inversion. which is why we've had it as a member variable in the past